### ISSUE TITLE
20180210写作机经预测——社会发展/生活

### ISSUE THREAD

**Issue by HurleyJames** [HurleyJames/IELTS_Writing#5]
It is generally agreed that making noise is an impolite behavior. Some people take a position that general public has the responsibility to control their noise strictly but others consider whether to make noise is a personal right. In my view, noise should be controlled depends on different conditions.
It is inevitable to make some noise in daily life. For example, in the final of World Cup, every family will rise to their feet and cheer wildly when their favorite player score in the final minute of the game. Likewise, the public will also cheer and even scream, making a series of high pitched sound, to welcome festivals like carnival or the New Year. People make noise in these situations in order to express their positive emotions, which is reasonable and acceptable. 
However, to control noise is necessary in most cases, both for individual and society. To individual, employees can increase their work efficiency in a quiet working environment. To much noise will distract people's attention so that they may not complete their urgent task on time. Besides, some medical research has proved that noise may cause negative influence on people's physical health and even result in some diseases, such as heart attack and high blood pressure. Therefore, it's better for people to keep away from noise for physical and mental health. From the social perspective, controlling noise reflects people's mutual care and attention. Only in this way can people build a harmonious social relationship, both in relatives and coworkers. If everyone did not make any unnecessary noise, society could be more peaceful. 
To sum up, although mush noise throughout our life is unavoidable, it should be controlled by the government since it may be harmful for personal health and social harmony.
